sdk/containerregistry/azure-containerregistry/azure/containerregistry/_container_repository_client.py

```diff
@@ -402,6 +532,26 @@ def set_tag_properties(self, tag, permissions, **kwargs):
         :type permissions: ContentPermissions
         :returns: :class:`~azure.containerregistry.TagProperties`
         :raises: :class:`~azure.core.exceptions.ResourceNotFoundError`
+
+        Example
```

Comment:
is there some sort of guideline on which samples should be inserted here vs. pointing to the samples folder? Was this just decided on due to length of sample?

Reply:
We don't have a great understanding of how customers are going to use the library, which means we have limited samples but I wanted to include how each client method would be used so this was my compromise.

Reply:
makes sense. Since you said things may change after the UX studies, I'm good with this